pageserver/src/http/routes.rs

```diff
@@ -969,7 +999,8 @@ pub fn make_router(
         .post("/v1/tenant", tenant_create_handler)
         .get("/v1/tenant/:tenant_id", tenant_status)
         .get("/v1/tenant/:tenant_id/size", tenant_size_handler)
-        .put("/v1/tenant/config", tenant_config_handler)
+        .put("/v1/tenant/config", update_tenant_config_handler)
+        .get("/v1/tenant/:tenant_id/config", get_tenant_config_handler)
```

Comment:
Much nicer this way than with ... Any way like the `/v1/tenant/config` (tenant id in the body) :+1:

Reply:
Yes, I'd change the `/v1/tenant/config` later too, as part of general config ergonomics improvements, but that comes later.